### PR TITLE
changed other to Other

### LINKED ISSUE #862: Policies Without Category
Hi,

While going through the policies over at https://kyverno.io/policies/, I noticed that there are some policies that don't appear under any **Policy Category**. I also noticed that it says there are 306 policies whereas when I select all the categories under **Policy Category**, it shows only 295 policies. I think grouping the non-categorized policies somewhere (possibly under the **Other** category) would be useful.

One example of this is: https://kyverno.io/policies/other/block-cluster-admin-from-ns/block-cluster-admin-from-ns/

I think the problem is that the `category` is set to `other` (lowercase) whereas the others in the **Other** category are set to `Other`

```yaml
metadata:
  name: block-cluster-admin-from-ns
  annotations:
    policies.kyverno.io/title: Block a Cluster Admin/s from CREATE/UPDATE/DELETE any object in a Namespace
    policies.kyverno.io/category: other
```

Here is a list of all policies that have `category: other` instead of `category: Other`:

```
other/verify-manifest-integrity/verify-manifest-integrity.yaml
other/update-image-tag/update-image-tag.yaml
other/scale-deployment-zero/scale-deployment-zero.yaml
other/restart-deployment-on-secret-change/restart-deployment-on-secret-change.yaml
other/require-unique-uid-per-workload/require-unique-uid-per-workload.yaml
other/record-creation-details/record-creation-details.yaml
other/label-nodes-cri/label-nodes-cri.yaml
other/generate-networkpolicy-existing/generate-networkpolicy-existing.yaml
other/cordon-and-drain-node/cordon-and-drain-node.yaml
other/concatenate-configmaps/concatenate-configmaps.yaml
other/block-cluster-admin-from-ns/block-cluster-admin-from-ns.yaml
```

You'll notice that these add up to 11, which covers the difference (306 - 295)

### PR DESCRIPTION
**Pull request by swastik959**
## Related Issue(s)
fixes #862 

## Description
changed other to Other

## Checklist

- [] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/sample_policy.yml
Comment (chipzoller) [inline .github/ISSUE_TEMPLATE/sample_policy.yml:1]: This is not a policy and should not be changed.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please see the [contribution guide](https://github.com/kyverno/policies#contribution), specifically the points about changing the `artifacthub-pkg.yaml` file when policies are changed.

**Comment by swastik959** [comment]
@chipzoller please review it now

**Review by chipzoller — CHANGES_REQUESTED** [review]
You only updated a single policy's corresponding `artifacthub-pkg.yml` file. Every one must be updated for each policy being updated.

**Comment by swastik959** [comment]
@chipzoller done

**Review by chipzoller — CHANGES_REQUESTED** [review]

File: other/block-cluster-admin-from-ns/artifacthub-pkg.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ readme: |
   In some cases we would want to block operations (CREATE/UPDATE/DELETE) of certain privileged users (i.e. cluster-admins), in a specific namespace.
           In this policy, Kyverno look for all user operations (`CREATE, UPDATE, DELETE`), on every object kind (Pod,Deployment,Route,Service,etc.), in the testnamespace namespace, and for the `clusterRole cluster-admin`. The `subject User testuser` is also mentioned so it won’t include all the cluster-admins in the cluster, but will be flexiable enough to apply only for a sub-group of the cluster-admins in the cluster.  
 annotations: 
-  policies.kyverno.io/category: other
+  policies.kyverno.io/category: Other
```
[the remark this answers is not in the document]
Comment (chipzoller) [reply other/block-cluster-admin-from-ns/artifacthub-pkg.yml:21]: `kyverno/category`

**Comment by swastik959** [comment]
@chipzoller please check it now

**Review by chipzoller — APPROVED** [review]

**Pull request merged**